docs/index.html

```diff
@@ -76,7 +79,7 @@ <h3>React</h3>
 
 								// Set the first todo as complete
 								const clone = this.state.todos.slice();
-								clone[0].complete = true;
+								Object.assign({}, clone[0], { complete: true });
```

Comment:
need `clone[0] =`. also might make more semantic sense to use `todos` instead of clone